src/corehost/cli/apphost/apphost.windows.cpp

```diff
@@ -76,12 +76,19 @@ namespace
             while (std::getline(ss, line, _X('\n'))){
                 const pal::string_t prefix = _X("The framework '");
                 const pal::string_t suffix = _X("' was not found.");
+				const pal::string_t custom_prefix = _X("  _ ");
```

Comment:
It seems you used Tabs for indentation - please make it consistent with the rest of the file (spaces).

Reply:
Seems to be the case for all .cpp/.h file changes in this PR.